stylesheets/_component.rules.scss

```diff
@@ -298,3 +298,306 @@ td.table__form .rules + .form__group:last-of-type {
+        padding: .5em;
+
+        @include respond-min($screen-tablet) {
+            padding: 0;
+        }
+    }
+
+    .rule-block__heading {
+        padding: 0;
+    }
+
+    .rule-block--if,
+    .rule-block--else-if {
+        background-color: lighten(#3f8ccc, 40);
+        border-color: #3f8ccc;
+        border-top-left-radius: 15px;
+
+        @include respond-min($screen-tablet) {
+            border-top-right-radius: 15px;
+        }
+
+        .rule-block__header {
+            background-color: #3f8ccc;
+            border-top-left-radius: 14px;
+        }
+
+        .remove-control {
+            color: color(white);
+
+            @include respond-min($screen-tablet) {
+                color: color(text);
+            }
+        }
+    }
+
+    .rule-block--else-if {
+        &::after {
+            border-top: 4px solid color(gray, lightest);
+            content: '';
+            display: block;
+            width: 24px;
+            height: 4px;
+            left: -25px;
+            position: absolute;
+            top: 35px;
+        }
+
+        .rule-block__header {
+            border-top: 0;
+        }
+    }
+
+    .rule-block--and {
+        background-color: lighten(#e8b750, 30);
+        border-color: #e8b750;
+        margin-top: 0;
+
+        .rule-block__header {
+            background-color: #f7d183;
+            color: color(black);
+        }
+    }
+
+    .rule-block--or {
+        background-color: lighten(#9d7ee3, 25);
+        border-color: #9d7ee3;
+
+        .rule-block__header {
+            background-color: #9d7ee3;
+        }
+    }
+
+    .rule-block--else {
+        background-color: lighten(#3f8ccc, 30);
+        border-color: #3f8ccc;
+
+        .rule-block__header {
+            background-color: #3f8ccc;
+        }
+
+        + .rule-block--return {
+            margin-left: 30px;
+            width: calc(100% - 30px);
+
+            &::before {
+                display: none;
+            }
+        }
+    }
+
+    .rule-block--return {
+        background-color: lighten(#7dbc8e, 30);
+        border-color: #7dbc8e;
+        border-bottom-left-radius: 15px;
+        border-bottom-right-radius: 15px;
+
+        .rule-block__header {
+            background-color: #7dbc8e;
+
+            @include respond-min($screen-tablet) {
+                border-bottom-left-radius: 14px;
+            }
+        }
+    }
+
+    .rule-block--else-return {
+        background-color: color(gray, lightest);
+        border-color: color(gray, dark);
+        border-radius: 15px;
+
+        .rule-block__header {
+            background-color: color(gray, dark);
+            border-top-left-radius: 14px;
+            color: color(white);
+
+            @include respond-min($screen-small) {
+                border-top-left-radius: 14px;
+                border-bottom-left-radius: 14px;
+                width: 120px;
+            }
+        }
+    }
+
+    .rule-block__controls {
+        margin-top: .5em;
```

Comment:
We've got a mix of em and px margins in these changes.

Reply:
**em** vs. **px** 🥊 *ding ding* 🔔

Reply:
The ems are overriding/matching form control styles